rdkit/Chem/UnitTestPeriodicTable.py

```diff
@@ -0,0 +1,29 @@
+# $Id$
```

Comment:
Please don't include the `$Id$` in new stuff. That's a leftover from the SVN days.